build.gradle

```diff
@@ -33,7 +33,7 @@ dependencies {
     implementation 'dev.zarr:jzarr:0.4.2'
     implementation 'info.picocli:picocli:4.7.5'
     implementation 'me.tongfei:progressbar:0.9.0'
-    implementation 'ome:formats-bsd:7.0.1'
+    implementation 'ome:formats-bsd:7.1.0'
     implementation 'com.glencoesoftware:bioformats2raw:0.8.0'
     implementation 'org.json:json:20230227'
```

Comment:
Should this be removed as it is now coming as a transitive dependency of `ome:formats-bsd` or do we want to control the version via Gradle?